src/test/java/com/iexec/worker/executor/TaskManagerServiceTests.java

```diff
@@ -1381,15 +1375,33 @@ void shouldNotCompleteSinceCannotRemoveResult() {
 
     //region abort
     @Test
-    void shouldAbortTask() {
+    void shouldNotifyErrorWhenRemainingContainers() {
```

Comment:
I don't like "notify" here. When I read this word, I imagine there is some sort of communication sent somewhere else. It doesn't seem to be the case here, so shouldn't we stick to a more common word?
```suggestion
    void shouldReturnErrorWhenRemainingContainers() {
```
Same issue with the following test method.

Reply:
new name pushed, see last commit :pray: